### DISCUSSION TITLE
Add reverse geocoding

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5885] [Feature Requests]
> It would be nice if we could upload a record with latitude and longitude and have Specify automatically assign or suggest a country, state, county based on the geocoordinates - I came across [this 2](https://opencagedata.com/guides/how-to-determine-which-country-coordinates-are-in) service which might be able to do this. It would be useful for offshore records that are difficult to determine if they fall within a political boundary or are in international waters.

**Reported By:** Paul Larson at FWRI ([Speciforum](https://discourse.specifysoftware.org/t/reverse-geocoding/1376))

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/reverse-geocoding/1376/2

**Comment by grantfitzsimmons** [comment]
This was liked by Zsuzsanna with NHMD

**Comment by grantfitzsimmons** [comment]
Liked by Nate Shoobs at Ohio Mollusks

**Comment by grantfitzsimmons** [comment]
+1 from Naturhistorisches Museum Bern
